### DISCUSSION TITLE
Custom Buttons download is corrupted

### DISCUSSION THREAD

**Discussion by CTFlorian** [arkypita/LaserGRBL#2485] [General]
I have tried all my decompression packages and every one says the file is corrupted.  Who do I report this to?

**Comment by arkypita** [comment]
That's because you don't need to uncompress it.
Simply follow the instructions you can read in the custom buttons download page: 
![Screenshot_2024-12-30-10-56-40-795_com opera browser](https://github.com/user-attachments/assets/5fd04464-4db1-4c1d-923e-f5c370372579)
